pkg/managed/testutils.go

```diff
@@ -0,0 +1,226 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
```

Comment:
not sure this is needed outside fo managed package.  If we find that is so then we should move it to util package.